### ISSUE TITLE
Support for mobile browsers

### ISSUE THREAD

**Issue by danfozzy** [DobyTang/LazyLibrarian#286]
Any plans to create a mobile friendly interface for LL?

**Comment by philborman** [comment]
Have a look at https://github.com/warlord0/lazylibrarian.bookstrap
Work in progress but not far off complete 
Magazine cover sizes need work, but the rest is looking really good

**Comment by danfozzy** [comment]
thats cool! will this be merged into this git?

**Comment by danfozzy** [comment]
Is there any reason why this can't be merged now? If there are any issues the user can go back and select the default interface?

**Comment by philborman** [comment]
You can merge yourself easily enough, just cd into 
/opt/LazyLibrarian/data/  (or wherever yours is installed), and git 
clone https://github.com/warlord0/lazylibrarian.bookstrap

Might get merged into main when complete, it's up to the author

**Comment by maxcore** [comment]
Bug with cover ? Very big view lol

**Comment by philborman** [comment]
yes, the author does know. Hopefully working soon.

**Comment by danfozzy** [comment]
@philborman I've done as you've explained, but I still don't have it, this is what I did:

root@Diskstation:/volume1/@appstore/LazyLibrarian/data# git clone https://github.com/warlord0/lazylibrarian.bookstrap
Cloning into 'lazylibrarian.bookstrap'...
remote: Counting objects: 165, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 165 (delta 4), reused 0 (delta 0), pack-reused 149
Receiving objects: 100% (165/165), 94.61 KiB | 0 bytes/s, done.
Resolving deltas: 100% (88/88), done.
Checking connectivity... done
root@Diskstation:/volume1/@appstore/LazyLibrarian/data# ls
bookstrap      bootbox.min.js       bootstrap-notify.min.js  images      js                       lazylibrarian-bs.js  README.md
bookstrap.css  bootstrap-notify.js  css                      interfaces  lazylibrarian.bookstrap  ll.ico
root@Diskstation:/volume1/@appstore/LazyLibrarian/data#

**Comment by philborman** [comment]
Might be better to wait a few days as there are big changes soon. Once it's
stable we might merge the files into the main dobytang one.

If you want to try sooner,

cp - r  lazylibrarian/bookstrap/*

into your /LazyLibrarian/data/ directory

Then bootstrap should appear as an extra item on the config menu

**Comment by danfozzy** [comment]
any advance on this? eager to see this merged!

**Comment by philborman** [comment]
Waiting on the original author.
The current version on his git site is a little behind the times because 
of interface changes in the base program. Extra config options etc.

Once it's all working in sync we can add it.

**Comment by danfozzy** [comment]
Do you have contact with the dev behind it? Been no updates on his git for a while?

**Comment by philborman** [comment]
No contact from him for a few weeks now,  but i have a fully working
version here. Not sure what the protocol would be for merging his code into
the main dobytang repository as its not my code or my repository ☺

**Comment by danfozzy** [comment]
@nncrypted ?

**Comment by nncrypted** [comment]
No idea about git protocol. I guess if you forked his work and added your own code to it then in theory you could merge it wherever you want. Since this is a open source community it shouldn't matter if you merged his into this one as long as he still gets credit for his work.
